bootstrap.ps1

```diff
@@ -11,6 +11,20 @@ Push-Location (Join-Path $PSScriptRoot "src/Simulation/Simulators")
     .\FindNuspecReferences.ps1
 Pop-Location
 
+# install prereqs for native builds
+if ($Env:ENABLE_NATIVE -ne "false") {
+    if (-not (Test-Path Env:AGENT_OS) -or ($Env:AGENT_OS.StartsWith("Win"))) {
+        choco install llvm
```

Comment:
these pre-reqs should be listed in the global README.md, or a point in the README about how to skip building QIR

Reply:
For now I suggest we have ENABLE_QIRRUNTIME environment variable which is set in the CI builds but not locally so by default people won't need to deal with the prereqs and the build at all or even be aware of it. And if someone decides to contribute to the QirRuntime, its own readme outlines the required build/test steps. As the project matures and becomes required part of qsharp-runtime we will promote the corresponding info to the top.